### ISSUE TITLE
Wine games specific config and display flexibility

### ISSUE THREAD

**Issue by dancaer69** [flightlessmango/MangoHud#36]
-When trying to use a config file in application's folder using wine, it doesn't load and the default config used instead. So seems that there is no way to use specific config file for an application which runs through wine. 

-Though I like a lot the look of this hud, I would like to have more control to displayed elements. As it is now, I didn't find a way to turn off CPU/GPU elements and graph. Seems that there is no option to display only FPS or CPU load for example. It also would be nice if have 3 modes(simple/full/no_display) and switch them from the shortcut key. Like Assassin's Creed Odyssey has.

**Comment by flightlessmango** [comment]
Sorry for the confusion but it's not properly documented yet.
Wine/Proton apps config files need to be prefix with `wine-`, for example `wine-appname.conf`

To only display FPS and CPU load you would use `MANGOHUD_CONFIG=frame_timing=0,gpu_stats=0`

**Comment by dancaer69** [comment]
I don't think I've understand. I tried to rename the config file to wine-MangoHud.conf, I also tried to rename it to game's name like: wine-Gamename.conf. But neather worked and alwayes the default config file loads.

**Comment by flightlessmango** [comment]
I'm not sure per app is working in the game folder, try adding the config to `$HOME/.config/MangoHud` and name it `appname.conf` and or `wine-appname.conf`

**Comment by dancaer69** [comment]
I tried it with 2 different games and it doesn't work too. It does work in native game(without wine), even without to change name of config file(just copy MangoHud.conf and edit it), but with wine always the default config loads. With wine only by using MANGOHUD_CONFIG I can change default configuration.

**Comment by flightlessmango** [comment]
So if I understand it correctly, if you put MangoHud.conf in a game folder it only works with native and not wine/proton?
also did you try like I suggested, to add it in .config and name it `wine-appname.conf` ?

**Comment by dancaer69** [comment]
For native games I tried only to copy MangoHud.conf on game's executable folder and edit it. I didn't try anything else because it works.
For wine games I tried all your suggestions and didn't work none of them. The only way works is if I use MANGOHUD_CONFIG directly, by using "export MANGOHUD_CONFIG=....." in a script before run wine command.

**Comment by Ahmed-E-86** [comment]
No, you need to put `wine-appname.conf` inside `$HOME/.config/MangoHud`

`MANGOHUD_CONFIG=.....` works only in this branch https://github.com/flightlessmango/MangoHud/tree/envvar

to build from this branch you need to write these commands in the terminal:

```
git clone --recurse-submodules https://github.com/flightlessmango/MangoHud.git
cd MangoHud
git checkout envvar
./build.sh install
```

**Comment by dancaer69** [comment]
As I wrote above I tried all suggestions. This includes yours. So, I already tried to putwine-appname.conf inside $HOME/.config/MangoHud  directory and it didn't work.

I just followed the installation instructions from here. I didn't build anything just clone and run install.sh. But MANGOHUD_CONF works for me.

**Comment by jackun** [comment]
Run it from terminal or if using Lutris, right click and select "Show logs". It should print every config mangohud tries to load. Search for "parsing/skipping config".

**Comment by dancaer69** [comment]
I get this messages:
`parsing config: /home/user/.local/share/MangoHud/MangoHud.conf [ ok ]
parsing config: /home/user/.config/MangoHud/MangoHud.conf [ ok ]
skipping config: /home/user/.config/MangoHud/wine-preloader.conf
skipping config: /opt/wine-staging/bin/MangoHud.conf
skipping config: /home/user/.config/MangoHud/wine-ACBSP.conf
`
Seems that it doesn't search in game's folder for config files but in wine's folder. I tried also to rename the file to "wine-preloader.conf" in /.config/MangoHud directory, but it still skip the file.

**Comment by jackun** [comment]
`/home/user/.config/MangoHud/wine-ACBSP.conf` is what you need if you want per-game config

**Comment by dancaer69** [comment]
Sorry, it wasn't clear in my previous comment. The file I renamed was the wine-ACBSP.conf. So, was there but it skipped.

**Comment by jackun** [comment]
Wait so it existed but was skipped? Hmm, file couldn't be opened then for some reason.

**Comment by dancaer69** [comment]
Sorry, this works. Seems that the name had some typo(maybe and extra space). After I copied the name from the terminal output worked. It didn't worked before because the games I tried had more that one executable which act like a launcher. 
Your suggestion about terminal output helped me find for which file and location need to look. I also look for terminal output when something goes wrong in an app, but because I use scripts to run wine games I have a lot of other messages and I missed the mangohud's messages.
Anyway with your help I found how it works for wine games. Thanks.
